mitosheet/mitosheet/tests/test_mito_analytics.py

```diff
@@ -8,8 +8,64 @@
 performing correctly
 """
 
+import sys
+from mitosheet.enterprise.mito_config import MITO_CONFIG_JUPYTER_LOG_SERVER_BATCH_INTERVAL, MITO_CONFIG_JUPYTER_LOG_SERVER_URL, MITO_CONFIG_VERSION
 from mitosheet.telemetry.telemetry_utils import PRINT_LOGS
+import pytest
+from unittest.mock import patch
+import os
+import json
+from mitosheet.tests.test_mito_config import delete_all_mito_config_environment_variables
+from mitosheet.tests.test_utils import create_mito_wrapper_with_data
+
+try:
+    from pandas import __version__ as pandas_version
+except:
+    pandas_version = 'no pandas'
+try:
+    # Format version_python as "3.8.5"
+    version_python = '.'.join([str(x) for x in sys.version_info[:3]])
+except:
+    version_python = 'no python'
 
 
 def test_not_printing_logs():
-    assert PRINT_LOGS is False
\ No newline at end of file
+    assert PRINT_LOGS is False
+
+def test_log_uploader():
```

Comment:
Can we test:
1. A long upload interval does not call .post at all
2. Multiple edit events all get posted
3. Errors
4. Mitosheet rendered (you can mock this one somehow)